packages/remix-architect/sessions/arcTableSessionStorage.ts

```diff
@@ -40,10 +44,10 @@ interface ArcTableSessionStorageOptions {
  *     _idx *String
  *     _ttl TTL
  */
-export function createArcTableSessionStorage({
+export function createArcTableSessionStorage<SD = SessionData, FD = SD>({
```

Comment:
I think we should have long & readable names instead of `SD` & `FD`

Reply:
Do you think `Data` and `FlashData` would be good enough?

Reply:
As long as it's clear what it's representing, I think it's good.

Reply:
PR updated :+1: